ci/ray_ci/doc/mock/__init__.py

```diff
@@ -0,0 +1,7 @@
+from ci.ray_ci.doc.mock.mock_module import MockClass
+from ci.ray_ci.doc.mock.mock_module import mock_function
+
+__all__ = [
+    "MockClass",
```

Comment:
this creates `ci.ray_ci.doc.mock.MockClass` as an alias for the full name `ci.ray_ci.doc.mock.mock_module.MockClass`